specification/resources/resource-manager/Microsoft.Resources/stable/2020-01-01/subscriptions.json

```diff
@@ -182,6 +182,55 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.Resources/checkResourceName": {
```

Comment:
Should we go back and update all existing `subscriptions.json` files for older API versions, or is this the only API version that supports this endpoint?